pkg/auth/ldaputil/query.go

```diff
@@ -137,6 +149,11 @@ func (o *LDAPQueryOnAttribute) buildAttributeQuery(attributeValue string,
 
 	filter := fmt.Sprintf("(&(%s)(%s))", o.Filter, specificFilter)
```

Comment:
unrelated, but just noticed this... are we double parenthesizing the `o.Filter` incorrectly?